### DISCUSSION TITLE
Praetor to join Akash Network Core team (at Overclock Labs) to accelerate provider growth

### DISCUSSION THREAD

**Discussion by lechenghiskhan** [akash-network/community#525] [Governance Proposals]
**Summary:**

Overclock Labs (OCL) and Praetor are proposing joining forces to accelerate growth of providers on Akash Network. If this proposal is accepted, the following will apply:

* [PraetorApp](https://akash.praetorapp.com/auth/login) will become fully open-source - enabling/ inviting contributions from the Akash community.
* OCL (Akash Core team), Akash Community and Praetor team will work together closely for a minimum of 12 months to continue to maintain PraetorApp and improve it per the strategy outlined [here](https://github.com/akash-network/community/blob/main/product-strategy/praetor-akash-provider-console-strategy.md).
* In the same structure as the [Cloudmos acquisition](https://github.com/orgs/akash-network/discussions/286), OCL will contribute 34,188 AKT (1/3rd) and request 68,376 AKT (2/3rd) from the Community Pool totaling 102,564 AKT, broken down as follows:
    * 46,620 AKT will be used for the acquisition of Praetor  
    * 55,944 AKT will be used to retain Jigar and Deval (co-founders of PraetorApp) for 12 months to continue maintain and advance PraetorApp

#

Overclock Labs (core team behind Akash Network) is proposing joining forces with the Praetor team to co-develop products to expand Akash Network. This collaborative acquisition is the result of weeks of discussions between both teams to agree on terms that further our shared goal of developing great open-source products for Akash Network.

The three points below provide an overview of the proposal, which are designed to ensure a smooth transition and support successful collaboration between Overclock Labs, the Akash Network community and Praetor.

### A Commitment to Open-Source

As a key part of the acquisition, Praetor will open-source its codebase to the Akash Network community - to align with the [rest of the OSS projects](https://github.com/akash-network) that enable Akash Network to function. By sharing code and opening the development process, developers from around the world can contribute directly to the growth of the Akash ecosystem.

### Collaborative Development

Overclock Labs and the Praetor team will work together closely for a minimum of 12 months to continue developing PraetorApp. During this period, we will actively collaborate, and share code, to eliminate redundancy and increase the speed of development. By collaborating closely, PraetorApp will be able to expand its capabilities faster, working closely with the core engineering team and the Akash Console team to improve user experience for Akash users and providers or compute.

### Community Pool Funding

Overclock Labs is requesting a total of 68,376 AKT from the Community Pool. This funding will be utilized to support the continued growth and development of Praetor along with fully open-sourcing the Praetor codebase. The AKT granted will vest over 12 months.

#
Everything related to Akash is fully open-source. This applies to the code that powers the network, but it also extends to the processes for governance and management. Contributors engage with Akash through a unique structure of working groups that meet publicly. Anyone from around the world can attend. This radically open structure allows for a level of collaboration that is not possible in closed-source development. 

Among this global community of developers, Praetor has emerged as a standout contributor. Their flagship application, PraetorApp, is widely used by providers to onboard onto the network - onboarding 10s of providers over the last 1-2 years. They have also moved quickly to ship critical provider tools and have been extremely committed to supporting the Akash community.

We think that this collaboration represents a win-win-win scenario for Akash Network’s community, Overclock Labs, and Praetor. By joining forces, Praetor will continue to make it easier and more intuitive to become and remain successful as a provider on Akash Network.

The primary goal of this acquisition is to ultimately deliver more value to our community, users and customers as we move towards cloud parity. With a shared codebase, we will be able to improve UX, ship new features quickly, reduce duplicate work and present a cohesive user experience to Akash users and customers. Together, we will accelerate the development of the world’s first operational Supercloud.

**Comment by SophoclesSummer** [comment]
This proposal was approved on 4/12/2024 by a 93% majority.
https://www.mintscan.io/akash/proposals/255
